kolibri/plugins/facility/assets/src/views/UserPage/index.vue

```diff
@@ -495,4 +495,9 @@
     overflow-x: auto;
   }
 
+  /deep/ .main-wrapper {
```

Comment:
I think my main concern here (and I think this was also what @marcellamaki was trying to express) is not that having a wider container is a bad thing, but using one of `/deep/` or `!important` would be a code smell, so doing both together is a code... stink?

We have already changed our overall width of page containers for much of the learn experience, so it doesn't seem like we're bound to that decision from ~2018.